packages/compiler-sfc/__tests__/compileScript.spec.ts

```diff
@@ -141,6 +141,17 @@ const myEmit = defineEmit(['foo', 'bar'])
       )
     })
 
+    test('should allow defineProps/Emit at the start of imports', () => {
```

Comment:
Maybe it's better to have a comment linking to the PR/issue here.

Reply:
Added a reference to #2740, following the convention that I saw in other tests (not the full link, just the number)